tests/integration/003_configuration_variable/main.tf

```diff
@@ -32,10 +32,3 @@ data "env0_configuration_variable" "tested1" {
   depends_on = [env0_configuration_variable.tested1]
 }
 
-output "tested1_value" {
```

Comment:
why did you remove this ?
If anything I'd like to see you add an enum variable here.

Reply:
2290182b
added enum check